src/Controls/src/Build.Tasks/nuget/buildTransitive/netstandard2.0/maui-blazor.aotprofile.txt

```diff
@@ -2698,8 +2803,11 @@ Methods:
 	System.Delegate System.Delegate:CreateDelegate (System.Type,System.Type,string)
 	System.Delegate System.Delegate:CreateDelegate_internal (System.Runtime.CompilerServices.QCallTypeHandle,object,System.Reflection.MethodInfo,bool)
 	System.Delegate System.Delegate:Remove (System.Delegate,System.Delegate)
+	System.Delegate System.Linq.Expressions.Compiler.LambdaCompiler:Compile (System.Linq.Expressions.LambdaExpression)
+	System.Delegate System.Linq.Expressions.Compiler.LambdaCompiler:CreateDelegate ()
```

Comment:
This looks like something is compiling `System.Linq.Expressions` at startup. We should probably investigate what that is, as it's probably slowing down startup.

Reply:
Oh this is only in the `maui-blazor` app, so it's probably something ASP.NET is doing. The `maui` app doesn't have `System.Linq.Expressions.Compiler`.

Reply:
That is new? Did ASP.NET start using LINQ in startup?

Reply:
I've always seen stuff that wasn't great for startup on Android in the `maui-blazor` template. But it was always code in 
ASP.NET.

Reply:
the numbers I posted were for maui and not maui-blazor.